pkgs/by-name/pe/perf_data_converter/package.nix

```
@@ -0,0 +1,63 @@
+{ lib
+, stdenv
+, buildBazelPackage
+, fetchFromGitHub
+, fetchpatch
+, bazel_6
+, jdk
+, elfutils
+, libcap
+}:
+
+buildBazelPackage rec {
+  pname = "perf_data_converter";
+  version = "unstable-2024-03-12";
+
+  src = fetchFromGitHub {
+    owner = "google";
+    repo  = "perf_data_converter";
+    rev   = "e1cfe1e7e5d8cf3b728a166bf02d4227c82801eb";
+    hash  = "sha256-Y3tBLH2jf1f28o6RK2inq9FulKc66qcqwKmxYdFC5tA=";
+  };
+
+  bazel = bazel_6;
+  bazelFlags = [
+    "--java_runtime_version=local_jdk"
+    "--tool_java_runtime_version=local_jdk"
+  ];
+
+  fetchAttrs = {
+    sha256 = "sha256-IauQ4zEn9YHppLgW+4XjPv4N5IZlEkp//tE/Dy7k28s=";
+  };
+
+  nativeBuildInputs = [
+    jdk
+  ];
+
+  buildInputs = [
+    elfutils
+    libcap
+  ];
+
+  removeRulesCC = false;
+
+  bazelBuildFlags = ["-c opt"];
+  bazelTargets = [ "src:perf_to_profile" ];
+
+  bazelTestTargets = ["src:all"];
+
+  buildAttrs = {
+    installPhase = ''
+      mkdir -p "$out/bin"
+      install bazel-bin/src/perf_to_profile "$out/bin"
+    '';
+  };
+
+  meta = with lib; {
+    description = "Tool to convert Linux perf files to the profile.proto format used by pprof. This allows to use pprof directly on perf.data output.";
```

```suggestion
    description = "Tool to convert Linux perf files to the profile.proto format used by pprof";
```

Be just one sentence.

---

Done.